code/__HELPERS/mobs.dm

```diff
@@ -486,3 +486,15 @@ GLOBAL_LIST_EMPTY(species_list)
 		chosen = pick(mob_spawn_meancritters)
 	var/mob/living/simple_animal/C = new chosen(spawn_location)
 	return C
+
+/proc/get_living_mobs()
+	. = list()
+	for(var/mob/living/player in GLOB.mob_list)
+		if(player.stat != DEAD && player.mind && is_station_level(player.z) && !isnewplayer(player) && !isbrain(player))
+			. |= player.mind
+
+/proc/get_living_crew()

```

Comment:
`get_human_station_minds()` or something

Reply:
Actually it shouldn't have minds in it, it should have mobs. So the code needs to be changed.